src/main/java/org/kitteh/irc/client/library/element/ISupportParameter.java

```diff
@@ -207,6 +207,31 @@
     }
 
     /**
+     * Represents support for EXTBAN.
+     */
+    interface ExtBan extends ISupportParameter {
+        /**
+         * Parameter name.
+         */
+        String NAME = "EXTBAN";
+
+        /**
+         * Gets the extban prefix.
+         *
+         * @return extban prefix if present
+         */
+        Optional<Character> getPrefix();
+
+        /**
+         * Gets the extbans.
+         *
+         * @return extbans
+         */
+        @Nonnull
+        List<ChannelMode> getExtBans();
```

Comment:
I don't think it really "aligns" with a channel mode any more than it aligns with a Pair (char,String). Definitely isn't representing channel modes.